styles.css

```diff
@@ -433,6 +478,65 @@ Desktop hidden stuff
   display: none;
 }
 
+/*
+Pop Up windows Styles
+*/
+.bg-container {
+  position: fixed;
+  padding: 20px 16px 26px;
+  height: 100%;
+  background: #c1c7d0;
+  opacity: 0;
+  pointer-events: none;
+  top: 0;
+  left: 0;
+  transition: opacity 0.3s ease;
+  overflow: scroll;
+}
+
+.mobile-pop {
```

Comment:
- [ ] The popup window is too big please consider giving it a smaller ```width``` and ```height``` so it is closer to the given design as possible

- Your design:
![Screenshot_20220512_185634](https://user-images.githubusercontent.com/25227022/168139367-d4ebd8e6-befa-46d3-8328-48fac7adbe7c.png)

- Figma design:
![Screenshot_20220512_185600](https://user-images.githubusercontent.com/25227022/168139409-fe0cdbe9-350a-4d10-a70d-c492a875e329.png)